client/state/data-layer/wpcom-http/pipeline/remove-duplicate-gets/index.js

```diff
@@ -19,20 +17,20 @@ const debug = debugFactory( 'calypso:data-layer:remove-duplicate-gets' );
  * @module state/data-layer/wpcom-http/optimizations/remove-duplicate-gets
  */
 
-/** @type {Map} holds in-transit request keys */
-const requestQueue = new Map();
+/** @type {Map<String,Object>} holds in-transit request keys */
+const requestLog = new Map();
 
 /**
  * Empties the duplication queue
  *
  * FOR TESTING ONLY!
  */
-export const clearQueue = () => {
+export const clearRequestLog = () => {
 	if ( 'undefined' !== typeof window ) {
```

Comment:
This would work, too:
```js
if ( process.env.NODE_ENV === 'test' ) {
    requestLog.clear();
    return;
}
throw new Error( '`clearQueue()` is not for use in production - only in testing!' );
```
The nice part about that is that Babel and Webpack remove the code inside if statement for production.

Reply:
Now, I'm wondering if you can wrap `export` statement in `if` clause :)